.github/workflows/test-msys2.yml

```diff
@@ -62,7 +65,7 @@ jobs:
               ${{ matrix.package }}-openjpeg2 \
```

Comment:
It is much nice to use the [pacboy `:p` way of installing packages](https://github.com/msys2/setup-msys2#pacboy), no need to worry about this "package" prefix...